packages/wasm/src/keys.test.ts

```diff
@@ -77,4 +79,16 @@ describe('keys', () => {
       }).not.toThrow();
     });
   });
+
+  describe('getIndexByAddress()', () => {
+    it('does not raise zod validation error', () => {
```

Comment:
Given it's an `Option<pb::AddressIndex>`, can you provide a case that returns `undefined`?